### DISCUSSION TITLE
Regarding Multi Language PDF data extraction

### DISCUSSION THREAD

**Discussion by Sathish36311** [jsvine/pdfplumber#1127] [Q&A]
How the characters will be extracted from different language of a pdf. How to handle that

**Comment by jsvine** [comment]
The characters will be extracted in whatever language the PDF encodes them in.

**Comment by Sathish36311** [comment]
But Im not getting the exact character which is visualised in the pdf, I get like 'a question mark inside a rhombus', but the actual character is micro. So how can I extract those as visualized (the exact one)

**Reply by jsvine** [reply]
In that case, it seems likely that there is a font-encoding issue with the PDF. One way to test whether this is fixable via `pdfplumber`: If you open the PDF in a standard PDF-viewing application, select the text, and paste it into a text editor, do you see the same problem, or do you get the text you'd expect?

**Reply by Sathish36311** [reply]
Yeah If I open the pdf in Adobe and copy paste the text, I get the exact text in the text editor.

**Reply by Sathish36311** [reply]
Is there any solution for that in pdfplumber

**Reply by jsvine** [reply]
If that's what you're seeing, the issue is likely with the library that `pdfplumber` uses to extract the low-level information from the PDFs: [`pdfminer.six`](https://github.com/pdfminer/pdfminer.six). Unfortunately, there's not much `pdfplumber` can do on its own to fix font-encoding issues.

**Reply by Sathish36311** [reply]
So can I handle that encoding issue in the pdfminer.six?

**Reply by Sathish36311** [reply]
I get the same unicode in pdfminer too. I get like '\u03bc', '\uf06d' of � (actual character is mu 'μ'). Is there any way to convert that Unicode to actual character @jsvine
